packages/spec/integration/esbuild-typescript/package.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Should we also run a jest test in this integration test, to demonstrate how it works?